openpype/hosts/houdini/api/lib.py

```diff
@@ -1054,3 +1054,72 @@ def add_self_publish_button(node):
     template = node.parmTemplateGroup()
     template.insertBefore((0,), button_parm)
     node.setParmTemplateGroup(template)
+
+
+def getSceneViewer():
+    """
+    return an instance of a visible viewport.
+    There may be many, some could be closed, any visible are current
+    """
+    panes = hou.ui.paneTabs()
+    panes = [x for x in panes if x.type() == hou.paneTabType.SceneViewer]
+    panes = sorted(panes, key=lambda x: x.isCurrentTab())
+    if panes:
+        return panes[-1]
+    else:
+        return None
+
+
+def sceneview_snapshoot(
```

Comment:
Never heard of snapshoot. Did you mean snapshot?
```suggestion
def sceneview_snapshot(
```